awx/main/tests/factories/tower.py

```diff
@@ -220,7 +220,7 @@ def create_job_template(name, roles=None, persisted=True, webhook_service='', **
     if 'organization' in kwargs:
         org = kwargs['organization']
         if type(org) is not Organization:
-            org = mk_organization(org, '%s-desc'.format(org), persisted=persisted)
+            org = mk_organization(org, '{}'.format(org), persisted=persisted)
```

Comment:
Why not just `(org, org, persisted=persisted)`?

Reply:
Yeah, that would work too.